<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QDialog" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1130</width>
    <height>798</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Register window</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>430</y>
     <width>257</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>The Sunday</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #030554;
font: 12pt &quot;The Sunday&quot;;</string>
   </property>
   <property name="text">
    <string>Confirm password :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>110</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>The Sunday</family>
     <pointsize>28</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 28pt &quot;The Sunday&quot;;
color: #030554;</string>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>350</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>The Sunday</family>
     <pointsize>14</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	font: 14pt &quot;The Sunday&quot;;
    color: #030554;
    border: 2px solid #01004c;
    border-radius: 10px;
    padding: 5px;
    background-color: white;
}
</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="passwordStrengthLabel">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>540</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="registerButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>630</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>The Sunday</family>
     <pointsize>14</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{ 
background-color:  #030554;
      color: white;
        font-size: 20px;
	font: 14pt &quot;The Sunday&quot;;
        border-radius: 30px;
        padding: 10px 20px
}</string>
   </property>
   <property name="text">
    <string>Sign Up</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>210</y>
     <width>140</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>The Sunday</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #030554;
font: 12pt &quot;The Sunday&quot;;</string>
   </property>
   <property name="text">
    <string>Username :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton_2">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>630</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>The Sunday</family>
     <pointsize>14</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{ 
background-color:  #030554;
      color: white;
        font-size: 20px;
	font: 14pt &quot;The Sunday&quot;;
        border-radius: 30px;
        padding: 10px 20px
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>320</y>
     <width>141</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>The Sunday</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #030554;
font: 12pt &quot;The Sunday&quot;;</string>
   </property>
   <property name="text">
    <string>Password :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>460</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>The Sunday</family>
     <pointsize>14</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	font: 14pt &quot;The Sunday&quot;;
    color: #030554;
    border: 2px solid #01004c;
    border-radius: 10px;
    padding: 5px;
    background-color: white;
}
</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>240</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>The Sunday</family>
     <pointsize>14</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	font: 14pt &quot;The Sunday&quot;;
    color: #030554;
    border: 2px solid #01004c;
    border-radius: 10px;
    padding: 5px;
    background-color: white;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1131</width>
     <height>801</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/LoginPhoto2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>770</y>
     <width>601</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>The Sunday</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #030554;
font: 9pt &quot;The Sunday&quot;;

</string>
   </property>
   <property name="text">
    <string>© 2025 – Tic Tac Toe Game | Embedded Systems Project | Supervised by Dr. Omar Nasr</string>
   </property>
  </widget>
  <zorder>label_6</zorder>
  <zorder>label_4</zorder>
  <zorder>label</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>passwordStrengthLabel</zorder>
  <zorder>registerButton</zorder>
  <zorder>label_5</zorder>
  <zorder>cancelButton_2</zorder>
  <zorder>label_3</zorder>
  <zorder>lineEdit_3</zorder>
  <zorder>lineEdit</zorder>
  <zorder>label_7</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
